arrows/mvg/applets/tests/CMakeLists.txt

```diff
@@ -4,3 +4,8 @@ add_test(NAME mvg:applets:track_features
   COMMAND kwiver track-features
             --config "${kwiver_test_data_directory}/track_features_testing.conf"
             "${kwiver_test_data_directory}/aphill_240_1fps_crf32.mp4")
+add_test(NAME mvg:applets:init_cameras_landmarks
```

Comment:
This is still timing out on the dashboards, probably because some of those machines run tests without threading.

How long does this take for you to run locally?

We could add `set_tests_properties(mvg:applets:init_cameras_landmarks PROPERTIES TIMEOUT 120)` below to let it run for 2 minutes and then see how long it actually takes on these dashboard machines.  Ultimately I think we do want to get it under 30 seconds.

You could try `initializer:mvg:essential_mat_estimator:vxl:num_ransac_samples = 50` in the config, that may still work and should be faster.  We could also cut the track file down further if needed.